compiler/rustc_typeck/src/check/generator_interior/drop_ranges/cfg_build.rs

```diff
@@ -209,6 +209,45 @@ impl<'a, 'tcx> DropRangeVisitor<'a, 'tcx> {
             self.drop_ranges.add_control_edge(self.expr_index + 1, self.expr_index + 1);
         }
     }
+
+    /// Break and continue expression targets might be another expression or a block,
+    /// but this analysis only looks at expressions. In case a break has a block as a
+    /// target, this will find the last expression in the block and return its HirId
+    /// instead.
+    fn find_target_expression(&self, hir_id: HirId) -> HirId {
```

Comment:
Comment/name could be improved: the fn *seems* to only find the target for breaks, but the name sounds generic, and the comment mentions both break and continue. Also, what is `hir_id`, is it the hir-id of the break? etc.